### ISSUE TITLE
[VPAT/HTML5] type attribute

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#5088]
Related to #5086 

W3C's validator is issuing the following warning:
> Warning: The type attribute is unnecessary for JavaScript resources.

This is in regards to `<script type='text/javascript' src='somefile.js'></script>`

Unlike earlier HTML, HTML 5 assumes scripts are javascript by default, and therefore the type attribute is unnecessary. 

My opinion is that this doesn't really matter and can be ignored.  However, as it is a prevalent validation warning, I am bringing it to attention to determine if anything should be done.  One (small) advantage to clearing the warning is it will improve on the validation report.
![Screen Shot 2020-02-28 at 7 31 51 PM](https://user-images.githubusercontent.com/12703624/75597245-6a9a1880-5a62-11ea-8314-87a4e6c9edfa.png)

Note that javascript is referenced throughout Submitty's `.twig` files.  There are many instances "to fix".

**Comment by elihschiff** [comment]
I solved this in the branch eli_5088 because it took me like 2min to fix. But I wont make a PR until this is discussed and we agree to fix this or not.
